### PR TITLE
Magic Link Login: Mk 1

### PR DESCRIPTION
**Pull request by jleandroperez**
### Details:
In this PR we're building the remote tools to invoke the new Authentication Endpoints.
Nothing can be tested **directly** at this point, as we're setting up the foundation for the new feature.

### Test
- [ ] Verify the Unit Tests still work
- [x] Verify you can still log into your Account!

Thank you!!

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: Simplenote/SimplenoteConstants.swift [hunk 1]
```diff
@@ -24,8 +24,16 @@ class SimplenoteConstants: NSObject {
...
 
+    /// Simplenote: Current Platform
+    ///
+    static let simplenotePlatformName = "macOS"
+
     /// URL(s)
     ///
+/// TODO: FIXME
+    static let loginRequestURL              = "https://magic-links-dot-simple-note-hrd.appspot.com/account/request-login" ////currentEngineBaseURL.appendingPathComponent("/account/request-login")
+/// TODO: FIXME
```
Comment (jleandroperez) [inline Simplenote/SimplenoteConstants.swift:35]: ⚠️ To be changed as soon as we deploy!!

File: Simplenote/SimplenoteConstants.swift [hunk 2]
```diff
@@ -24,8 +24,16 @@ class SimplenoteConstants: NSObject {
...
     static let currentEngineBaseURL = "https://app.simplenote.com" as NSString
 
+    /// Simplenote: Current Platform
+    ///
+    static let simplenotePlatformName = "macOS"
+
     /// URL(s)
     ///
+/// TODO: FIXME
+    static let loginRequestURL              = "https://magic-links-dot-simple-note-hrd.appspot.com/account/request-login" ////currentEngineBaseURL.appendingPathComponent("/account/request-login")
```
Comment (jleandroperez) [inline Simplenote/SimplenoteConstants.swift]: ⚠️ To be changed as soon as we deploy!!

File: Simplenote/SimplenoteConstants.swift [hunk 3]
```diff
@@ -24,8 +24,16 @@ class SimplenoteConstants: NSObject {
...
+/// TODO: FIXME
+    static let loginCompletionURL           = "https://magic-links-dot-simple-note-hrd.appspot.com/account/complete-login" ////currentEngineBaseURL.appendingPathComponent("/account/complete-login")
```
Comment (jleandroperez) [inline Simplenote/SimplenoteConstants.swift]: ⚠️ To be changed as soon as we deploy!!

**Review by charliescheer — APPROVED** [review]
looks good

**Comment by jleandroperez** [comment]
Thank youuu!!

**Pull request merged**